piece_of_cake.py

```diff
@@ -0,0 +1,14 @@
+# Arguments:
+# prices - dictionary[key: item's name, value: price for 100g]
+# optionals - list of ingredients to ignore.
+# ingredients - dictionary[ingredient_name : amount in grams]
+# Return value - float
+# This functions calculates the total price for wanted ingredients
+def get_recipe_price(prices, optionals=[], **ingredients):
```

Comment:
Write documentation in this format
![Capture](https://user-images.githubusercontent.com/13467749/159859320-1f3faca5-02dd-40c9-8574-500083624390.JPG)